web/packages/teleport/src/Notifications/Notification.story.tsx

```diff
@@ -133,19 +132,21 @@ export const NotificationsList = () => <ListComponent />;
 NotificationsList.parameters = {
   msw: {
     handlers: [
-      rest.get(cfg.api.notificationsPath, (req, res, ctx) =>
-        res.once(ctx.json(mockNotificationsResponseFirstPage))
```

Comment:
Here `res.once` was necessary, as there's another handler below for `cfg.api.notificationsPath`.